sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -266,6 +327,9 @@ SymbolicName=EVENT_BAD_ACCOUNT_NAME
 Language=English
 No backslash is in the account name.
 .
+Language=Polish
+W nazwie konta brak znaku kreski ułamkowej odwróconej.
```

Comment:
W nazwie konta brak znaku ukośnik odwrotny. (just bumped into this repo found this line slightly confusing).

Reply:
When I was translating the file, I was using netevent.dll from Win2k as a reference, so the messages would make it look like Windows (with changing Microsoft/Windows to ReactOS).